### PR TITLE
manual processing menu is now closed in world map

### PR DESCRIPTION
**Pull request by Blaussh**

### CODE REVIEW

File: src/app/@ansyn/menu-items/tools/tools/tools.component.ts [hunk 1]
```diff
@@ -24,6 +24,11 @@ export class ToolsComponent implements OnInit, OnDestroy {
 		.map((tools: IToolsState) => tools.flags)
 		.distinctUntilChanged();
 
+	public imageProcessingDisabled$: Observable<boolean> = this.store.select(toolsStateSelector)
+		.map((state: IToolsState) => !!state.flags.get(toolsFlags.imageProcessingDisabled))
```
Comment (shaharido1) [inline src/app/@ansyn/menu-items/tools/tools/tools.component.ts]: pluck

File: src/app/@ansyn/menu-items/tools/tools/tools.component.ts [hunk 2]
```diff
@@ -24,6 +24,11 @@ export class ToolsComponent implements OnInit, OnDestroy {
...
+		.distinctUntilChanged()
+		.filter(Boolean);
+
```
Comment (shaharido1) [inline src/app/@ansyn/menu-items/tools/tools/tools.component.ts]: do

File: src/app/@ansyn/menu-items/tools/tools/tools.component.ts [hunk 3]
```diff
@@ -71,6 +76,9 @@ export class ToolsComponent implements OnInit, OnDestroy {
 			this.subMenu$.subscribe(),
 			this.flags$.subscribe(_flags => {
 				this.flags = _flags;
+			}),
+			this.imageProcessingDisabled$.subscribe(() => {
```
Comment (shaharido1) [inline src/app/@ansyn/menu-items/tools/tools/tools.component.ts]: subscribers.push() . onDestroy... unsub

File: src/app/@ansyn/ansyn/app-effects/effects/tools.app.effects.ts [hunk 4]
```diff
@@ -112,7 +112,15 @@ export class ToolsAppEffects {
 	onActiveMapChangesSetOverlaysFootprintMode$: Observable<any> = this.actions$
 		.ofType(MapActionTypes.TRIGGER.ACTIVE_MAP_CHANGED)
 		.withLatestFrom(this.store$.select(mapStateSelector), (action, mapState: IMapState) => MapFacadeService.activeMap(mapState))
-		.map((activeMap: CaseMapState) => new SetActiveOverlaysFootprintModeAction(activeMap.data.overlayDisplayMode));
+		.mergeMap((activeMap: CaseMapState) => {
+			const actions = [];
+			actions.push(new SetActiveOverlaysFootprintModeAction(activeMap.data.overlayDisplayMode));
```
Comment (shaharido1) [inline src/app/@ansyn/ansyn/app-effects/effects/tools.app.effects.ts]: init with action

File: src/app/@ansyn/menu-items/tools/tools/tools.component.ts (hunk 1 above)
Comment (Blaussh) [reply src/app/@ansyn/menu-items/tools/tools/tools.component.ts]: done

File: src/app/@ansyn/menu-items/tools/tools/tools.component.ts (hunk 2 above)
Comment (Blaussh) [reply src/app/@ansyn/menu-items/tools/tools/tools.component.ts]: done

File: src/app/@ansyn/menu-items/tools/tools/tools.component.ts (hunk 3 above)
Comment (Blaussh) [reply src/app/@ansyn/menu-items/tools/tools/tools.component.ts]: already implemented in code

File: src/app/@ansyn/ansyn/app-effects/effects/tools.app.effects.ts (hunk 4 above)
Comment (Blaussh) [reply src/app/@ansyn/ansyn/app-effects/effects/tools.app.effects.ts]: done

**Pull request merged**